packages/hash/frontend/src/pages/shared/ory-kratos.ts

```diff
@@ -0,0 +1,191 @@
+import {
+  Configuration,
+  V0alpha2Api,
+  SelfServiceError,
+  SelfServiceLoginFlow,
+  SelfServiceRegistrationFlow,
+  SelfServiceBrowserLocationChangeRequiredError,
+  ErrorAuthenticatorAssuranceLevelNotSatisfied,
+  NeedsPrivilegedSessionError,
+} from "@ory/client";
+import { isUiNodeInputAttributes } from "@ory/integrations/ui";
+import { AxiosError } from "axios";
+import { NextRouter, useRouter } from "next/router";
+import {
+  DependencyList,
+  Dispatch,
+  SetStateAction,
+  useEffect,
+  useState,
+} from "react";
+
+type SelfServiceFlow = SelfServiceLoginFlow | SelfServiceRegistrationFlow;
+
+export const oryKratosClient = new V0alpha2Api(
+  new Configuration({
+    /**
+     * Directly connecting to kratos (using "http://127.0.0.1:4433") would prevent the
+     * CRSF token from being set as an HTTP-Cookie, because the browser cannot send or
+     * receive cookies via the browser `fetch` method.
```

Comment:
Have added the clarifying wording in https://github.com/hashintel/hash/pull/889/commits/025f8de77f61b89c12b6896dcec5ebc8c3c9db43. I think it'd be good to capture here as it will help us remember why we have `/api/ory`.